frame/staking/src/pallet/mod.rs

```diff
@@ -431,6 +437,27 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
+	/// Paginated exposure of a validator at given era.
+	///
+	/// This is keyed first by the era index to allow bulk deletion, then the stash account and
+	/// finally the page.
+	///
+	/// It is removed after `HISTORY_DEPTH` eras.
+	/// If stakers hasn't been set or has been removed then empty exposure is returned.
+	#[pallet::storage]
+	#[pallet::getter(fn paged_eras_stakers)]
+	#[pallet::unbounded]
+	pub type PagedErasStakers<T: Config> = StorageNMap<
```

Comment:
todo: create a wrapper struct to access eras stakers, eras stakers clipped, eras stakers paged.

Reply:
Yes, I do some similar things in the multi-block branch as well: a private storage items with public getters.